### DISCUSSION TITLE
pyMOR School 2021

### DISCUSSION THREAD

**Discussion by renefritze** [pymor/pymor#1360] [Announcements]
We are very happy to invite you to the third pyMOR School, which will
take place from 4-8 October 2021 
at the Mathematics Münster Cluster of Excellence Germany.
https://school.pymor.org

This school is targeted at anyone interested to apply model order
reduction in their work using pyMOR (https://pymor.org). The school
will consist of both introductory lectures to model order reduction
and pyMOR as well as programming sessions. Participants are encouraged
to bring their own problems to work on. Basic knowledge of Python and
packages from the SciPy stack (NumPy, SciPy, matplotlib) is assumed.
For participants without any prior experience with Python, we have
linked introductory material on the school homepage.

You can register online before 3 September 2021. For details, see:
https://school.pymor.org/register

**Comment by yewalenikhil65** [comment]
Hey @renefritze 
That's exciting news. 
However, considering current travel bans(due to COVID) from certain parts of the world, is it possible that pyMOR school  2021 sessions be made available online also??

**Reply by renefritze** [reply]
[Yes](https://school.pymor.org/about.html#covid)! We're planning to live stream lectures.
